### PR TITLE
Pradnya - modified end interview code and changed interview page layout

### PR DESCRIPTION
**Pull request by pradnya-barve**

### CODE REVIEW

File: .github/workflows/docker-backend-fastapi.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+name: Docker
```
Comment (Suraj1089) [inline .github/workflows/docker-backend-fastapi.yml:1]: Remove ai-interviewer folder.

File: server/src/routes/interviews.ts [hunk 2]
```diff
@@ -18,4 +19,22 @@ router.get("/candidate/:id", auth, listInterviewsCandidate);
 router.get("/hr/:id", auth, listInterviewsHR);
 router.get("/:id", getMeeting);
 
+router.get("/:id/endtime", async (req, res) => {
```
Comment (Suraj1089) [inline server/src/routes/interviews.ts:22]: move the logic in controllers

**Review by Suraj1089 — CHANGES_REQUESTED** [review]

File: server/src/routes/interviews.ts [hunk 3]
```diff
@@ -10,4 +11,22 @@ router.get("/candidate/:id", auth, listInterviewsCandidate);
...
```
Comment (rasikaghadge) [inline server/src/routes/interviews.ts:14]: Opened an issue for this improvement. Hence, merging this PR.

File: server/src/routes/interviews.ts (hunk 2 above)
Comment (rasikaghadge) [reply server/src/routes/interviews.ts:14]: Opened an issue for this improvement. Merging this PR.

**Pull request merged**